lib/raw_net_capture.rb

```diff
@@ -1,7 +1,7 @@
 # encoding: utf-8
 
 class RawNetCapture < StringIO
```

Comment:
Does it make sense to alter `RawNetCapture` in this way? `RawNetCapture` doesn't really make any assumptions about the underlying protocol, so not sure it really makes sense to talk about 'transactions' for it